cmd/version.go

```diff
@@ -59,7 +61,7 @@ Examples:
 		} else {
 			release, err := util.GetLatestRelease(ctx)
 			if err != nil {
-				logger.Fatal("%s", err)
+				errsystem.New(errsystem.ErrGeneric, err).ShowErrorAndExit()
```

Comment:
review other uses of this command and try and match the pattern. you will likely need to add a new errorcodes.yaml and rerun go generate to reference it